### DISCUSSION TITLE
03_computer_vision_exercise

### DISCUSSION THREAD

**Discussion by AhmedYusri54** [mrdbourke/pytorch-deep-learning#932] [Q&A]
Hello can you guys help me on this
`
test_preds = []
model_2.eval()
with torch.inference_mode():
  for X_test, y_test in tqdm(fashion_mnist_train_dataloader):
    y_logits = model_2(X_test.to(device))
    y_pred = torch.softmax(y_logits, dim=1)
    y_labels = torch.argmax(y_pred, dim=1)
    test_preds.append(y_labels)
test_preds = torch.cat(test_preds).cpu()
test_preds[:10], len(test_preds) 
`
when i run this the output be like this 
`(tensor([3, 4, 9, 0, 8, 3, 9, 6, 8, 7]), 60000)`
but it must be like this 
`(tensor([9, 2, 1, 1, 6, 1, 4, 6, 5, 7]), 10000)`
so what is the error in my code ?
